Library/Formula/rethinkdb.rb

```diff
@@ -75,6 +73,11 @@ def plist; <<-EOS.undent
     </plist>
     EOS
   end
+
+  test do
+    shell_output("#{bin}/rethinkdb create -d test")
```

Comment:
Ok, [It is a rethinkdb's bug](https://github.com/rethinkdb/rethinkdb/commit/8a0d68f76c3c772b5be6970f08a8ad75a957733e). Either way, this PR is ready to be merged.